tasks.py

```diff
@@ -19,8 +19,13 @@
 from invoke import Collection, task as invoke_task
 from invoke.exceptions import Exit
 
-# Override built-in print function with rich's pretty-printer function
-from rich import print  # pylint: disable=redefined-builtin
+try:
+    # Override built-in print function with rich's pretty-printer function, if available
+    from rich import print  # pylint: disable=redefined-builtin
+
+    RICH_PRINT = True
```

Comment:
This is a construct used in Ansible, we don't have to use the same name, but perhaps resonates and putting forward for consideration?

```
    HAS_LIB = True
```

Feel free to resolve as I understand we do not have to match what Ansible does.

Reply:
Just to make sure I understand, you're suggesting `HAS_RICH` rather than `RICH_PRINT`? I'm fine with that.

Reply:
They literally just use HAS_LIB each time, but I think that HAS_RICH or HAS_$LIB is better.